src/IECoreGL/Primitive.cpp

```diff
@@ -273,7 +273,7 @@ void Primitive::render( State *state ) const
 	
 }
 
-const Shader::Setup *Primitive::shaderSetup( const Shader *shader, State *state ) const
+const Shader::Setup *Primitive::shaderSetup( const Shader *shader, State *state, bool vertexCsActive ) const
 {
 	for( ShaderSetupVector::const_iterator it = m_shaderSetups.begin(), eIt = m_shaderSetups.end(); it != eIt; it++ )
```

Comment:
This search is now broken - it cannot know whether the setup it finds was created with `vertexCsActive == true` or not, and can return the wrong one.